pynbody/sph/__init__.py

```diff
@@ -214,13 +232,23 @@ def get_value(self, d, h=1):
 
         return f / (math.pi * h ** 3)
 
+    @staticmethod
+    def wendlandC2(d,h):
+
+        if d < 2:
+            f = (1. - (d / 2.))**4 * (2. * d + 1)
+        else:
+            f = 0
+
+        return (21. * f) / (16. * math.pi * h ** 3)
+
 
 class Kernel2D(Kernel):
 
-    def __init__(self, k_orig=Kernel()):
+    def __init__(self, type='spline'):
```

Comment:
The parameter passed here should indeed be a `Kernel` object, and an appropriate subclass can then be passed if one wants to use a Wendland kernel